superset/cli.py

```diff
@@ -260,10 +261,30 @@ def import_dashboards(path: str, recursive: bool, username: str) -> None:
     "--dashboard-file", "-f", default=None, help="Specify the the file to export to"
 )
 @click.option(
-    "--print_stdout", "-p", is_flag=True, default=False, help="Print JSON to stdout"
+    "--print_stdout", "-p", is_flag=True, default=False, help="Print JSON to stdout",
 )
-def export_dashboards(dashboard_file: str, print_stdout: bool) -> None:
+def export_dashboards(
+    dashboard_file: Optional[str], print_stdout: bool = False
+) -> None:
     """Export dashboards to JSON"""
+    if app.config["DEFAULT_FEATURE_FLAGS"].get("VERSIONED_EXPORT"):
+        from superset.dashboards.commands.export import ExportDashboardsCommand
+        from superset.models.dashboard import Dashboard
+
+        g.user = security_manager.find_user(username="admin")
+
+        dashboard_ids = [id_ for (id_,) in db.session.query(Dashboard.id).all()]
```

Comment:
You can drop the `()` if you want.

Reply:
I think without the `()` it's easy to overlook what's happening:

```python
dashboard_ids = [id_ for id_, in db.session.query(Dashboard.id).all()] 
```

Adding parentheses make the intent more clear, IMHO.

Reply:
```
[dashboard.id for dashboard in db.session.query(Dashboard.id).all()]
```
is clearer, but no strong opinions about it

Reply:
@dpgaspar do you mean?

```python
[dashboard.id for dashboard in db.session.query(Dashboard).all()]
```

That's a more expensive query (`SELECT * FROM dbs` instead of `SELECT id FROM dbs`), no?

Reply:
yes, but no, I mean the way I wrote it that would generate a `SELECT id from dbs` but it's just a detail, if you feel more comfortable unpacking the tuples the way you wrote got ahead.